tests/by-util/test_stdbuf.rs

```diff
@@ -24,6 +25,18 @@ fn test_permission() {
         .stderr_contains("Permission denied");
 }
 
+#[cfg(feature = "feat_external_libstdbuf")]
+#[test]
+fn test_permission_external_missing_lib() {
```

Comment:
@Ecordonnier wdyt ? :)

Reply:
I think this needs to be written differently long-term:

- one corner-case is that this will fail in case the user ran "make install" before running this test
- testing "Permission denied" while building with feat_external_libstdbuf adds more value than skipping "test_permission" and testing for the "missing libstdbuf" error message
- one way to fix this is by implementing the same logic than in GNU coreutils: stdbuf should search for libstdbuf.so first in the path where stdbuf is currently located, and then in LIBSTDBUF_DIR. that way it would be easy to provide a symlink to libstdbuf.so for testing purpose (without requiring make install / root access)
- if you need to merge it now please add a big TODO

Reply:
yeah I think so...